### PR TITLE
ip_scopes update

### PR DESCRIPTION
**Pull request by n-marton**
This modification prevents ip_scopes plugin to return 'tunl' or 'docker' type interfaces' ipaddress as privateaddress.

### CODE REVIEW

**Comment by tas50** [comment]
Seems pretty reasonable. +1

File: lib/ohai/plugins/ip_scopes.rb [hunk 1]
```diff
@@ -51,7 +51,15 @@ def private_addr?(address)
     address.to_ip.scope =~ /PRIVATE/
   end
 
-  def tunnel_iface?(interface)
+  def ppp_iface?(interface)
     interface["type"] == "ppp"
   end
+
+  def tunel_iface?(interface)
```
Comment (btm) [inline lib/ohai/plugins/ip_scopes.rb]: why the change from tunnel to tunel? is this a typo in multiple places?

**Comment by n-marton** [comment]
fixed tunnel vs tunel stuff
